Outreach (destination) documentation

mohan-h opened the pull request:
Initial commit

djwendo commented:
Thank you for creating this PR. We are reviewing the documentation and will update shortly.

Tracking internally: https://jira.corp.adobe.com/browse/PLAT-102543

mohan-h committed:
Outreach (destination) documentation

Initial commit

vfalon commented on help/destinations/catalog/crm/outreach.md:
```diff
@@ -0,0 +1,208 @@
+
+## Overview {#overview}
+
+[Outreach](https://www.outreach.io/) is a Sales Execution Platform with the most B2B buyer-seller interaction data in the world and significant investments in proprietary AI technologies to translate sales data into intelligence, Outreach helps revenue organizations automate sales engagement and act on revenue intelligence to improve their efficiency, predictability, and growth.
```
Line 11: Full stop after "translate sales data into intelligence".

vfalon commented on help/destinations/catalog/crm/outreach.md:
```diff
@@ -0,0 +1,208 @@
+
+## Prerequisites {#prerequisites}
+
+### Prerequisites in Experience Platform {#prerequisites-in-experience-platform}
```
I would rephrase the headings as` Experience Platform prerequisites` and `Outreach prerequisites`.

vfalon commented on help/destinations/catalog/crm/outreach.md:
```diff
@@ -0,0 +1,208 @@
+
+#### You need to have an Outreach account {#prerequisites-account}
+
+Go to the Outreach [sign in](https://accounts.outreach.io/users/sign_in) page to register and create an account, if you do not have one already. Also see support [page](https://support.outreach.io/hc/en-us/articles/207238607-Claim-Your-Outreach-Account).
```
See the Outreach [support page](https://support.outreach.io/hc/en-us/articles/207238607-Claim-Your-Outreach-Account) for more details.

vfalon commented on help/destinations/catalog/crm/outreach.md:
```diff
@@ -0,0 +1,208 @@
+
+Go to the Outreach [sign in](https://accounts.outreach.io/users/sign_in) page to register and create an account, if you do not have one already. Also see support [page](https://support.outreach.io/hc/en-us/articles/207238607-Claim-Your-Outreach-Account).
+
+Refer to Adobe's documentation for [Segment Membership Details schema field group](https://experienceleague.adobe.com/docs/experience-platform/xdm/field-groups/profile/segmentation.html?lang=en) if you need guidance on segment statuses.
```
Line 35: why is this mentioned here? Shouldn't this be mentioned in the Experience Platform prerequisites section?

Also, you are linking to a page within the AEP docs repo, so please use a relative link to the Markdown file instead of the public Experience League page.

vfalon commented on help/destinations/catalog/crm/outreach.md:
```diff
@@ -0,0 +1,208 @@
+
+#### Setup custom field labels
+
+Outreach supports custom fields for [prospects](https://support.outreach.io/hc/en-us/articles/360001557554-Outreach-Prospect-Profile-Overview). Refer to [How To add a custom field in Outreach](https://support.outreach.io/hc/en-us/articles/219124908-How-To-Add-a-Custom-Field-in-Outreach) for additional guidance. For ease of identification it is recommended to manually update the label's to their c ...
```
Line 48: labels instead of label's.

vfalon commented on help/destinations/catalog/crm/outreach.md:
```diff
@@ -0,0 +1,208 @@
+
+Outreach supports custom fields for [prospects](https://support.outreach.io/hc/en-us/articles/360001557554-Outreach-Prospect-Profile-Overview). Refer to [How To add a custom field in Outreach](https://support.outreach.io/hc/en-us/articles/219124908-How-To-Add-a-Custom-Field-in-Outreach) for additional guidance. For ease of identification it is recommended to manually update the label's to their c ...
+
+Outreach settings page for prospects displaying custom fields.
```
Please enclose Outreach in [!DNL ] tags throughout the doc.

vfalon commented on help/destinations/catalog/crm/outreach.md:
```diff
@@ -0,0 +1,208 @@
+
+> [!NOTE]
+>
+> When updating prospects the label names are not used they are for ease of identification only.
```
Please rephrase this note. Something like "Label names are not used when updating prospects. They are only used for ease of identification.".

vfalon commented on help/destinations/catalog/crm/outreach.md:
```diff
@@ -0,0 +1,208 @@
+
+|Target Identity|Description|Considerations|
+|---|---|---|
+| OutreachId |Custom Outreach identifier that supports mapping of any identity.|Mandatory. You can send any [identity](../../../identity-service/namespaces.md) to the [!DNL Outreach] destination, as long as you map it to the `OutreachId`. |
```
Is OutreachId the name of the variable? If so, please enclose it in code tags. If it is shown as Outreach ID in the UI, please format it accordingly.

mohan-h replied:
It is an id within Outreach, however to make it easier for the end user within AEP its called OutreachId.

vfalon commented on help/destinations/catalog/crm/outreach.md:
```diff
@@ -0,0 +1,208 @@
+
+To connect to this destination, follow the steps described in the [destination configuration tutorial](../../ui/connect-destination.md). In the configure destination workflow, fill in the fields listed in the two sections below.
+
+![Catalog](../../assets/catalog/crm/outreach/catalog.png)
```
This screen has five highlights on it, but users might not understand the order those are shown in. Maybe add some numbering next to them (1. Destinations, 2.Catalog, etc.).

vfalon commented on help/destinations/catalog/crm/outreach.md:
```diff
@@ -0,0 +1,208 @@
+*  **[!UICONTROL Username]**: Your Outreach account email.
+*  **[!UICONTROL Password]**: Your Outreach account password.
+
+If the details provided are valid, the UI displays a **Connected** status with a green check mark, you can then proceed to the next step.
```
Full stop after "green check mark".

vfalon commented on help/destinations/catalog/crm/outreach.md:
```diff
@@ -0,0 +1,208 @@
+
+To correctly send your audience data from Adobe Experience Platform to the Outreach destination, you need to go through the field mapping step. Mapping consists of creating a link between your Experience Data Model (XDM) schema fields in your Platform account and their corresponding equivalents from the target destination. To correctly map your XDM fields to the Outreach destination fields, follo ...
+
+1. In the Mapping step, click **[!UICONTROL Add new mapping]**, you will see a new mapping row on the screen.
```
Line 134: "In the [UICONTROL Mapping] step".

vfalon commented on help/destinations/catalog/crm/outreach.md:
```diff
@@ -0,0 +1,208 @@
+1. In the Mapping step, click **[!UICONTROL Add new mapping]**, you will see a new mapping row on the screen.
+![Add new mapping](../../assets/catalog/crm/outreach/add-new-mapping.png)
+
+1. In the select source field window, when selecting the source field choose the **[!UICONTROL Select identity namespace]** category and add the mappings desired.
```
Line 137: "[!UICONTROL Select source field] window"

vfalon commented on help/destinations/catalog/crm/outreach.md:
```diff
@@ -0,0 +1,208 @@
+1. In the select source field window, when selecting the source field choose the **[!UICONTROL Select identity namespace]** category and add the mappings desired.
+![Source mapping](../../assets/catalog/crm/outreach/source-mapping.png)
+
+1. In the select target field window, select the target field and choose the **[!UICONTROL Select identity namespace]** category and add the mappings desired.
```
Steps 3&4 should be one single step with 2 bullet points, one showing the namespace mapping, one showing the custom attributes mapping. Please reorganize this step around this structure. Something like this:

In the [!UICONTROL Select target field] screen, select the target field type:
* **[!UICONTROL Select identity namespace]**: select this option to map your source field to an identity namespace from the list.
* **[!UICONTROL Select custom attributes]**: select this option to map your source field to a custom attribute that you define in the [!UICONTROL Attribute name] field.

vfalon commented on help/destinations/catalog/crm/outreach.md:
```diff
@@ -0,0 +1,208 @@
+1. For custom attributes, in the select target field window, select the target field and choose the **[!UICONTROL Select custom attributes]** category, Next provide the desired target attribute name and add the mappings desired.
+![Target mapping using LastName](../../assets/catalog/crm/outreach/target-mapping-lastname.png)
+
+1. For instance, you could add the following mapping between your XDM profile schema and your [!DNL Outreach] instance:
```
Line 146: This step is an example that should be included in the previous step.

vfalon commented on help/destinations/catalog/crm/outreach.md:
```diff
@@ -0,0 +1,208 @@
+
+1. For instance, you could add the following mapping between your XDM profile schema and your [!DNL Outreach] instance:
+
+    ||XDM Profile Schema|[!DNL Outreach] Instance| Mandatory|
```
What should the user understand from the Mandatory column? Is the Outreach ID identity mandatory mapping for everyone? If it is, we should call this out in the mapping step.  Also, the Attributes example doesn't have any mandatory column value. Should it be Yes/No?

mohan-h replied:
_What should the user understand from the Mandatory column? Is the Outreach ID identity mandatory mapping for everyone? If it is, we should call this out in the mapping step._ 
Added additional notes as suggested,

_Also, the Attributes example doesn't have any mandatory column value. Should it be Yes/No?_
It depends on the attributes that the end user wants to update. Hence there is no input provided. Have reworded, trust it would work.

vfalon commented on help/destinations/catalog/crm/outreach.md:
```diff
@@ -0,0 +1,208 @@
+
+When performing the [Schedule segment export](../../ui/activate/activate-segment-streaming-destinations.html?lang=en#scheduling) step you must manually map Platform segments to the custom field attribute in Outreach.
+
+To do this, select each segment, then enter the corresponding numeric value which corresponds to the *custom`N`* attribute from Outreach in the **[!UICONTROL Mapping ID]** field.
```
Let's spell out an example of this mandatory mapping ID. When reading this section, my understanding was that Mapping ID should have a value of custom1/custom2/custom3/etc. It doesn't look like that is the case from the screenshot.

mohan-h replied:
Thanks for highlighting have reworded.

vfalon reviewed (changes requested):
Thanks for opening the PR. There are a few changes that should be addressed before we can approve it. See my comments in the review.

vfalon commented on help/destinations/catalog/crm/outreach.md:
```diff
@@ -0,0 +1,208 @@
+
+|Target Identity|Description|Considerations|
+|---|---|---|
+| OutreachId |Custom Outreach identifier that supports mapping of any identity.|Mandatory. You can send any [identity](../../../identity-service/namespaces.md) to the [!DNL Outreach] destination, as long as you map it to the `OutreachId`. |
```
Let's add an example of an Outreach ID. What format does it have? What kind of mapping does it support? What can users map to an OutreachID?

mohan-h replied:
Thanks, have reworded.

vfalon commented on help/destinations/catalog/crm/outreach.md:
```diff
@@ -0,0 +1,208 @@
+
+When performing the [Schedule segment export](../../ui/activate/activate-segment-streaming-destinations.html?lang=en#scheduling) step you must manually map Platform segments to the custom field attribute in Outreach.
+
+To do this, select each segment, then enter the corresponding numeric value which corresponds to the *custom`N`* attribute from Outreach in the **[!UICONTROL Mapping ID]** field.
```
Let's add a separate section explaining how to obtain this mapping ID, step by step, with screenshots from the Outreach UI.

mohan-h replied:
Have updated the content, the rewording should hopefully fix this explanation discrepancy. 
P.s. We already have screengrab's in pre-requisites explaining creation of custom fields.

mohan-h committed:
Updates against review

Updates against review https://github.com/AdobeDocs/experience-platform.en/pull/93/#pullrequestreview-1061757002

mohan-h committed:
Format updates

vfalon commented on help/destinations/catalog/crm/outreach.md:
```diff
@@ -0,0 +1,217 @@
+
+To connect to this destination, follow the steps described in the [destination configuration tutorial](../../ui/connect-destination.md). In the configure destination workflow, fill in the fields listed in the two sections below.
+
+![Catalog](../../assets/catalog/crm/outreach/catalog.png)
```
This screenshot looks pretty cluttered. I would remove it completely, since this step is standard across all destinations and is described in the article you already linked to.

mohan-h replied:
If we remove it the user might not specifically choose the connector under the CRM category. Have updated with fewer highlights and no numbering (which is why I had avoided it in the first place).

mohan-h replied:
Have removed the screenshot.

vfalon commented on help/destinations/catalog/crm/outreach.md:
```diff
@@ -0,0 +1,217 @@
+1. In the "[!UICONTROL Mapping]" step, click **[!UICONTROL Add new mapping]**, you will see a new mapping row on the screen.
+![Add new mapping](../../assets/catalog/crm/outreach/add-new-mapping.png)
+
+1. In the "[!UICONTROL Select source field]" window, when selecting the source field choose the **[!UICONTROL Select identity namespace]** category and add the mappings desired.
```
Line 137: "[...] and add the desired mappings.".

vfalon commented on help/destinations/catalog/crm/outreach.md:
```diff
@@ -0,0 +1,217 @@
+1. In the "[!UICONTROL Select source field]" window, when selecting the source field choose the **[!UICONTROL Select identity namespace]** category and add the mappings desired.
+![Source mapping](../../assets/catalog/crm/outreach/source-mapping.png)
+
+1. In the "[!UICONTROL Select target field]" window, select the target field 
```
Line 140: In the "[!UICONTROL Select target field]" window, select the type of target field that you want to map the Outreach ID to.

mohan-h replied:
Have done the change but would read odd. Suggest to not explicitly mention Outreach Id as subsequent bullets cover not just the Identity namespace mapping but also other custom attributes for different fields.

vfalon reviewed (changes requested):
Documentation looks good. Just a few minor comments.

mohan-h committed:
Updates post review

vfalon commented on help/destinations/catalog/crm/outreach.md:
```diff
@@ -0,0 +1,217 @@
+1. In the "[!UICONTROL Select source field]" window, when selecting the source field choose the **[!UICONTROL Select identity namespace]** category and add the desired mappings.
+![Source mapping](../../assets/catalog/crm/outreach/source-mapping.png)
+
+1. In the "[!UICONTROL Select target field]" window, select the type of target field that you want to map the Outreach ID to.
```
Line 140: Enclose Outreach ID in DNL tags.

vfalon commented on help/destinations/catalog/crm/outreach.md:
```diff
@@ -0,0 +1,217 @@
+
+### Experience Platform Prerequisites {#prerequisites-in-experience-platform}
+
+Before activating data to the [!DNL Outreach] destination, you must have a [schema](/help/xdm/schema/composition.html), a [dataset](https://experienceleague.adobe.com/docs/platform-learn/tutorials/data-ingestion/create-datasets-and-ingest-data.html?lang=en), and [segments](https://experienceleague.adobe.com/docs/platform-learn/tutorials/segments/create-segments.html?lang=en) created in [!DNL Expe ...
```
Please run a search for "html" in this page and replace all HTML links with relative links to the the Markdown file in the repo, where needed. There are still instances of html pages.

mohan-h committed:
Updates post review

mohan-h committed:
Merge branch 'AdobeDocs:main' into outreach-dest

vfalon commented on help/destinations/catalog/crm/outreach.md:
```diff
@@ -0,0 +1,219 @@
+
+### Experience Platform Prerequisites {#prerequisites-in-experience-platform}
+
+Before activating data to the [!DNL Outreach] destination, you must have a [schema](/help/xdm/schema/composition.html), a [dataset](https://experienceleague.adobe.com/docs/platform-learn/tutorials/data-ingestion/create-datasets-and-ingest-data.html?lang=en), and [segments](https://experienceleague.adobe.com/docs/platform-learn/tutorials/segments/create-segments.html?lang=en) created in [!DNL Expe ...
```
Line 25: schema URL is linking to HTML. Please change to markdown file.

vfalon commented on help/destinations/catalog/crm/outreach.md:
```diff
@@ -0,0 +1,219 @@
+![[!DNL Outreach] custom fields defaults](../../assets/catalog/crm/outreach/outreach-custom-fields.png)
+
+[!DNL Outreach] settings page for prospects displaying custom fields with *user-friendly* labels matching to the segment names. You can view the segment status on the prospect page against these labels.
+![[!DNL Outreach] custom fields with labels](../../assets/catalog/crm/outreach/outreach-custom-field-labels.png)
```
One thing that applies to all destinations. To improve readability and accessibility for all our docs, we have been asked to use clearer alt-text for images. Please review all the alt-text in the page and update it to a form similar to this:

"Outreach UI screenshot showing custom fields with associated labels."

Use clear sentences that describe as accurate as possible what is shown in the image. This is especially useful for users who are unable to see the image and/or use screen readers.

vfalon commented on help/destinations/catalog/crm/outreach.md:
```diff
@@ -0,0 +1,219 @@
+
+The [!DNL Outreach] API is rate-limited on a per-user basis, with a fixed limit of 10,000 requests per one-hour period. If you encounter this limit you will encounter a `429` response with a message similar to *`You have exceeded your permitted rate limit of 10,000; please try again at 2017-01-01T00:00:00.`*
+
+Refer to the [[!DNL Outreach] documentation](https://api.outreach.io/api/v2/docs#:~:text=%3A00%22%20%7D%5D%20%7D-,Rate%20Limiting,requests%20per%20one%2Dhour%20period1) for additional details.
```
Line 219: That URL does not look right. I'm guessing it should just be a link to the home page.

vfalon reviewed (changes requested):
Left a couple more comments.

vfalon commented on help/destinations/catalog/crm/outreach.md:
```diff
@@ -0,0 +1,219 @@
+
+| Item | Type | Notes |
+---------|----------|---------|
+| Export type | **[!UICONTROL Profile-based]** | <ul><li>You are exporting all members of a segment, together with the desired schema fields *(for example: email address, phone number, last name)*, according to your field mapping.</li><li> Platform segment statuses are exported to [!DNL Outreach] by specifying their corresponding custom field attribute in [!DNL Outreach], within the **[!UICONTROL ...
```
On line 74 we need to rephrase the second bullet point:

- Platform segment statuses are exported to [!DNL Outreach] by entering their [!DNL Outreach] custom field attribute in the [!UICONTROL Mapping ID] field, during the [!UICONTROL Schedule segment export] step.

mohan-h replied:
There is a section #schedule-segment-export-example should I also link to it ?

vfalon replied:
Yes, please link to that section, but also rephrase the bullet point.

mohan-h committed:
Updates post review

mohan-h committed:
Updates

vfalon commented on help/destinations/catalog/crm/outreach.md:
```diff
@@ -0,0 +1,219 @@
+1. In the [!UICONTROL Mapping] step, click **[!UICONTROL Add new mapping]**, you will see a new mapping row on the screen.
+![Platform UI screenshot showing how to Add new mapping](../../assets/catalog/crm/outreach/add-new-mapping.png)
+
+1. In the [!UICONTROL Select source field] window, when selecting the source field choose the **[!UICONTROL Select identity namespace]** category and add the desired mappings.
```
Line 137: remove the "when selecting the source field" part. It's redundant.

vfalon commented on help/destinations/catalog/crm/outreach.md:
```diff
@@ -0,0 +1,219 @@
+        |`person.name.firstName`|`firstName`|
+        |`person.name.lastName`|`lastName`|
+
+1. An example using these mappings is shown below:
```
Line 158: this is not a step, it's an example. Please either indent it so it falls under the example above, or remove the numbering.

vfalon commented on help/destinations/catalog/crm/outreach.md:
```diff
@@ -0,0 +1,219 @@
+
+The [[!DNL Outreach] documentation](https://api.outreach.io/api/v2/docs/) has details on [Error Responses](https://api.outreach.io/api/v2/docs#error-responses) which you can use to debug any issues.
+
+### Limits
```
The limits section is not an "Additional resource". Please move this after the prerequisites section, as a `Guardrails` H2 heading.

vfalon commented on help/destinations/catalog/crm/outreach.md:
```diff
@@ -0,0 +1,219 @@
+
+## Prerequisites {#prerequisites}
+
+### Experience Platform Prerequisites {#prerequisites-in-experience-platform}
```
Sentence case for headings, please.

vfalon commented on help/destinations/catalog/crm/outreach.md:
```diff
@@ -0,0 +1,219 @@
+
+Refer to Adobe's documentation for [Segment Membership Details schema field group](/help/xdm/field-groups/profile/segmentation.md) if you need guidance on segment statuses.
+
+### Outreach Prerequisites {#prerequisites-destination}
```
Sentence case for headings.

vfalon reviewed (changes requested):
Please see my comments, as some of the previous ones seem to have been missed.

vfalon commented on help/destinations/catalog/crm/outreach.md:
```diff
@@ -0,0 +1,219 @@
+1. Monitor the segment summary and ensure that the count of profiles corresponds to the count created within the segment.
+![Platform UI screenshot showing Segment summary.](../../assets/catalog/crm/outreach/segment.png)
+
+1. Log in to the [!DNL Outreach] website, then navigate to the **[!DNL Apps]** > **[!DNL Contacts]** page and check if the profiles from the segment have been added. You will also notice the segment status from Experience Platform has been updated against the corresponding custom field attribute *(identified through the friendly label name)* that was provided in the Platform **Mapping ID** field  ...
```
Line 196: I think I already mentioned this in my previous comments. There is no [!UICONTROL Activate destination] UI element.

Please rephrase the section as below:

"Log in to the [!DNL Outreach] website, then navigate to the **[!DNL Apps]** > **[!DNL Contacts]** page and check if the profiles from the segment have been added. You can see that each segment status in [!DNL Outreach] was updated with the corresponding segment status from Platform, based on the [!UICONTROL Mapping ID] value provided during the segment scheduling step."

mohan-h committed:
Merge branch 'AdobeDocs:main' into outreach-dest

mohan-h committed:
Updated post review

mohan-h committed:
Updated post review
